settings.gradle.kts

```diff
@@ -1,5 +1,6 @@
 plugins {
     id("com.gradle.enterprise") version "3.14"
+    id("org.gradle.toolchains.foojay-resolver-convention") version "0.6.0"
```

Comment:
One last question: I don't see this plugin being used. Do we really need this? I thought Gradle has Java toolchains by default.

Reply:
Yes, in the past (7.x) it was, now not defining it is deprecated: https://github.com/mockito/mockito/pull/3064/commits/25958c56d3270c66b7a4669ad5cbab80b8f02edd